NFIQ2/NFIQ2Algorithm/src/tool/nfiq2_ui_refresh.cpp

```diff
@@ -730,6 +728,56 @@ NFIQ2UI::printHeader(
 	}
 }
 
+std::tuple<std::string, std::string>
+NFIQ2UI::parseModel(NFIQ2UI::Arguments arguments)
+{
+
+	std::string propFilePath;
+
+	if (arguments.flags.model == "") {
+		// Check common places for directory containing model
+		// If in Current Directory
+		if (BE::IO::Utility::fileExists("nfiq2rf.txt")) {
+			propFilePath = ".";
+		}
+		// Unix
+		else if (BE::IO::Utility::fileExists(
+			     "/usr/local/share/nfiq2/nfiq2rf.txt")) {
+			propFilePath = "/usr/local/share/nfiq2/";
+		}
+		// Windows
+		else if (BE::IO::Utility::fileExists(
+			     "C:\\Program Files\\nfiq2\\nfiq2rf.txt")) {
```

Comment:
I think this will change to "NFIQ 2" and "NFIQ 2 x64." We'll have to see how things shake out in the packaging branch.